src/main/java/seedu/algobase/logic/parser/AlgoBaseParser.java

```diff
@@ -176,12 +176,15 @@ public Command parseCommand(String userInput) throws ParseException {
 
         // UI
         case SwitchTabCommand.COMMAND_WORD:
+        case SwitchTabCommand.SHORT_COMMAND_WORD:
```

Comment:
Not sure if this is a good practice or not.

Reply:
Hmm why not?